conda_libmamba_solver/shards_subset.py

```diff
@@ -85,18 +107,20 @@ class Node:
     package: str = ""
     channel: str = ""
     visited: bool = False
+    shard_url: str = ""
 
     def to_id(self) -> NodeId:
-        return NodeId(self.package, self.channel)
+        return NodeId(self.package, self.channel, self.shard_url)
 
 
 @dataclass(order=True, eq=True, frozen=True)
 class NodeId:
     package: str
     channel: str
+    shard_url: str = ""
```

Comment:
Is the NodeId creation in the `neighbors` function ok with this change?

Reply:
Yes, shard_url may be the default empty string.